code/src/components/Survey.js

```diff
@@ -0,0 +1,206 @@
+import React, { useState } from 'react';
+
+import FirstPage from 'components/FirstPage';
+import UserFeelings from 'components/UserFeelings';
+import UserTeam from 'components/UserTeam';
+import UserAnswers from './UserAnswers';
+import UserConfirm from './UserConfirm';
+import UserInput from './UserInput';
+import UserAloneOrBuddy from './UserAloneOrBuddy';
+import UserHours from './UserHours';
+
+const Survey = () => {
+  const [counter, setCounter] = useState(0);
+  const [userName, setUserName] = useState('');
+  const [userFeelings, setUserFeelings] = useState('');
+  const [userTeam, setUserTeam] = useState('');
+  const [userBuddy, setUserBuddy] = useState('');
+  const [userAloneOrBuddy, setUserAloneOrBuddy] = useState('');
+  const [userHours, setUserHours] = useState('');
+
+  // This creates a counter. We use it to navigate into the survey and display the questions
+  const handleCounterButtonClick = (shouldIncrease) => {
+    if (shouldIncrease && counter < 10) {
+      setCounter(counter + 1)
+    } else if (!shouldIncrease && counter > 0) {
+      setCounter(counter - 1)
+    }
+  }
+  // This will reset the survey
```

Comment:
I totally forgot this step in my own code. It  old choices are just overwritten when user starts over but i think this is much better! great